rules/opentitan.bzl

```diff
@@ -226,7 +226,7 @@ def opentitan_binary(
 def verilator_params(
         rom = "//sw/device/boot_rom:boot_rom_verilator_scr",
         otp = "//hw/ip/otp_ctrl/data:rma_image_verilator",
-        tags = ["verilator"],
+        tags = ["verilator", "cpu:4"],
```

Comment:
Do we need this if we have `--local_test_jobs` set?

Reply:
I view this as the proper way to set a kind of test to require 4 cores as the verilator tests do. In the future allocating 4 cores to a different kind of test could become problematic because it's too few or too many and I wanted to at least start laying the groundwork for making this specific to verilator tests and allocating different specific numbers to different kinds of tests.

This doesn't appear to work (I still get timeouts with this set), but I hope to get some time to chase this down and fix it and that with the fix, the .bazelrc can be undone so we're not setting something global that should be specific to verilator tests.